packages/create-toolpad-app/src/templates/indexPage.ts

```diff
@@ -0,0 +1,46 @@
+import { SupportedRouter } from '../types';
+
+const indexPage = (authEnabled: boolean, routerType: SupportedRouter) => {
```

Comment:
Can't we just give all templates the same options as an object? That way we only have to create that once and can pass it to each template.